rust/src/ike/ikev1.rs

```diff
@@ -136,6 +136,11 @@ pub fn handle_ikev1(
                         &tx.hdr.ikev1_header.vendor_ids,
                         &tx.hdr.ikev1_transforms,
                     );
+
+                    if state.ikev1_container.server.transforms.len() > 1 {
```

Comment:
Should we set the event only once we cross the threshold ?
Something like `state.ikev1_container.server.transforms.len() <= 1 ` and `tx.hdr.ikev1_transforms.len() + state.ikev1_container.server.transforms.len()  > 1` before calling update ?

Reply:
yes, good idea